tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,448 @@
+---
+title: An Introduction to Geospatial Analysis
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-10-05
+---
+
+[Geometries - Open In Code Editor](https://code.earthengine.google.com/d2f8fc328c747296c610f0aafb15407a)
+
+[Features - Open In Code Editor](https://code.earthengine.google.com/1cc3ba5b1a6820c4f03804ead3123400)
+
+[Images - Open In Code Editor - Geometries](https://code.earthengine.google.com/2ba5e30e1178134ca60e59fca36ec925)
+
+We will introduce types of geospatial data, using these data on the Earth Engine platforms, and discuss a host of functionality to visualize and analyze them. This workshop was originally created for a workshop during Yale-NUS Data 2.0 hackathon and then updated for Yale GIS Day 2018. 
+
+## Introduction
+Collection, visualization, and analysis of geographical or spatial data.
+
+## Data types
+
+- Vector data represent lat-long coordinates
+
+- Raster data comprises of pixels with associated values
+
+---
+
+
+- Points
+
+![Points](points.png)
+
+- Lines
+
+![Lines](line-vector.png)
+
+- Polygons
+
+![Polygons](polygon-vector.png)
+
+- Raster layers/bands
+
+![Raster](map-algebra.png)
+
+
+---
+
+## Google Earth Engine platform
+
+[Introductory video](https://www.youtube.com/watch?v=gKGOeTFHnKY&feature=youtu.be, "Video")
+
+[Code Editor](https://code.earthengine.google.com/ "Earth Engine Code Editor")
+
+- Cloud-based platform for planetary scale geospatial analysis
+- Uses Google's computational resources to reduce processing time
+- Massive archive of remote sensing data 
+- 200+ public datasets 
+- greater than 4000 new images every day 
+- greater than 5 million images 
+- greater than 5 petabytes of data
+
+Source: Google Earth Engine User summit
+
+---
+
+![Code Editor (Source: developers.google.com)](ee-editor.png)
+
+---
+
+## Basic Functions
+
+### Declaring variables
+```javascript
+var varName = Containerforvariabletype(variable name); 
```

Comment:
Please remove var_ prefix and use standard lower camelcase for variable names (`convexGeo`, not `convex_geo or ConvexGeo`) here any throughout

Reply:
Done.